media-gfx/flameshot/flameshot-0.9.0.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop cmake xdg-utils
```

Comment:
You could just inherit *xdg.eclass*, and remove `pkg_post*` phases from this ebuild. Just to clean it up, but your choice.

Reply:
Will do